python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -329,26 +326,17 @@ def __deepcopy__(self, memo=None):
         # Due to problems on Hammercloud due to uncopyable object lets
         # explicitly stop these objects going anywhere near the __deepcopy__
 
-        cls = type(self)
-        c = Job.__new__(cls)
+        c = Job.__class__()
```

Comment:
i'll check but isn't this the same as just doing Job() which this should probably be. i was just trying to explicitly avoid the proxy